src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -488,11 +488,34 @@ public boolean openPairingWindowWithPINCallback(
         deviceControllerPtr, devicePtr, duration, iteration, discriminator, setupPinCode, callback);
   }
 
-  /* Shutdown all cluster attribute subscriptions for a given device */
-  public void shutdownSubscriptions(long devicePtr) {
-    shutdownSubscriptions(deviceControllerPtr, devicePtr);
+  public int getFabricIndex() {
+    return getFabricIndex(deviceControllerPtr);
   }
 
+  /* Shutdown all subscriptions */
```

Comment:
Should be in Javadoc format:

```
/**
 * Shuts down all active subscriptions.
 *
 * @param schedule if true, asynchronously schedules a shutdown instead of blocking
 */
```

Same for others.

Reply:
rollback the version without schedule and update the comments. thanks